Sources/Plasma/FeatureLib/pfMetalPipeline/plMetalDevice.cpp

```diff
@@ -344,7 +346,10 @@ void plMetalDevice::BeginNewRenderPass()
 
             // if we need postprocessing, output to the main pass texture
             // otherwise we can go straight to the drawable
```

Comment:
Is this comment still accurate (vs the updated one added below)?

Reply:
I think the comment is still accurate. I'll add a tad bit more language to clarify.